### DISCUSSION TITLE
Scheduled Github Action stops working if last committer deletes their accounts

### DISCUSSION THREAD

**Discussion by johanwulf** [community/community#109354] [Actions] [answered]
### Select Topic Area

Bug

### Body

I noticed this issue when merging my personal and work account. My previous work account was the latest committer of the workflows, and now when I deleted that account, the scheduled workflows are not running any more. If this is intended, I think there should be a warning displayed or a mail sent to other active users of the repository.

Thanks!

**Comment by sr-nu** [comment]
I  don't know why this is closed, without properly addressing the issue.

**Reply by johanwulf** [reply, marked as answer]
This behaviour is documented in the workflows documentation now, see https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule
